webcompat/static/css/development/components/comment.css

```diff
@@ -161,7 +166,6 @@
 
   /* QR */
   .wc-Comment-qr {
-    position:absolute;
     z-index: 1;
-    right: 0;
+    margin: -6px 16px 0 auto;
```

Comment:
Hehe! Thanks for pointing out! 

Made the fix. Had to retain `margin-left: auto` in `wc-Comment-qr` for right alignment.